pkg/dispatchcli/cmd/manage.go

```diff
@@ -0,0 +1,164 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"path"
+
+	"github.com/spf13/cobra"
+	"github.com/vmware/dispatch/pkg/dispatchcli/i18n"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/mitchellh/go-homedir"
+	restclient "k8s.io/client-go/rest"
+)
+
+var (
+	manageShort = `Manage Dispatch configurations.`
+	manageLong  = `Manage Dispatch configurations.`
+
+	manageExample = i18n.T(`
+# Enable Dispatch bootstrap mode while specifying bootstrap user and associated public key
```

Comment:
The public key is not necessary until the bootstrap user happens to be a service-account. You can modify this as follows :-

```
# Enable Dispatch bootstrap mode by specifying a valid bootstrap user (for Open ID Connect based authentication)
dispatch manage --enable-bootstrap-mode --bootstrap-user user@example.com

# Enable Dispatch bootstrap mode by specifying a bootstrap user and an associated public key (for JWT based authentication)
dispatch manage --enable-bootstrap-mode --bootstrap-user bootstrap-user --public-key ./bootstrap-user.key.pub
````

Reply:
sure, will do